manage.py

```diff
@@ -67,5 +67,17 @@ def seed_db():
     User('admin@test.com', '1234', '5415551234', [], 'admin').save()
 
 
+@manager.command
+def twillio_test():
+    """Quick sms test to make sure twillio is functioning as expected."""
+    from _15thnight.twilio_client import send_sms
+
+    with app.app_context():
+        # This default test number will force twillio to fail, but still be
+        # a valid api call
+        phone_number = app.config.get("TWILIO_TEST_NUMBER", "+15555555555")
```

Comment:
This should take in the phone number from the command line, then fall back to the config.